src/style/index.css

```diff
@@ -232,6 +232,12 @@ select {
   appearance: none;
 }
 
+select#theme option
```

Comment:
I reckon this could be rolled out to all options, not just the theme picker.

```suggestion
option {
```

Reply:
Yep, that's even better :)